util/generate_dict/generate_dicom_dict.py

```diff
@@ -82,8 +82,8 @@ def write_dict(fp, dict_name, attributes, tag_is_string):
         entry_format = f"{{Tag}}: {tag_content}"
 
     fp.write(f"\n{dict_name} = {{\n    ")
-    fp.write(",  # noqa\n    ".join(entry_format.format(**attr) for attr in attributes))
-    fp.write("  # noqa\n}\n")
+    fp.write(",\n    ".join(entry_format.format(**attr) for attr in attributes))
+    fp.write("\n}\n")
```

Comment:
Will the removals here affect the [dict](https://github.com/pydicom/pydicom/blob/main/src/pydicom/_dicom_dict.py) files after they're regenerated?

Reply:
Yes, they will affect these files after they're regenerated. The current lines won't be too long, except for a single line in one of the files. For that single line and to make this future-proof, I have instead added to `pyproject.toml`:
```toml
"src/pydicom/_*_dict.py" = ["E501"]
"src/pydicom/uid.py" = ["E501"]
```